packages/types-algosdk/index.d.ts

```diff
@@ -555,11 +555,26 @@ export interface SSCParams {
   'local-state-schema': SSCSchemaConfig;
 }
 
+export interface AssetParams {
+  creator: string;
+  total: number;
+  decimals: number;
+  'default-frozen': string;
+  'unit-name': string;
+  name: string;
+  url: string;
+  'metadata-hash': string;
+  manager: string;
+  reserve: string;
+  freeze: string;
+  clawback: string;
+}
+
 export interface AccountAssetInfo {
   amount: number;
   'asset-id': number;
   creator: string;
-  'is-frozen': boolean;
+  'is-frozen': string;
```

Comment:
why this is changed to string?

Reply:
I checked this in js-algorand repo.
https://github.com/algorand/js-algorand-sdk/blob/a5309ee57dddbf6f9db5f95dc8a82eb1ae03c326/tests/cucumber/steps/steps.js#L2092